imports/plugins/core/catalog/server/no-meteor/utils/createCatalogProduct.js

```diff
@@ -108,7 +108,7 @@ export async function xformProduct({ collections, product, shop, variants }) {
         variantInventory = {
           canBackorder: canBackorder(variantOptions),
           inventoryAvailableToSell: variant.inventoryAvailableToSell || 0,
-          inventoryInStock: variant.inventoryQuantity || 0,
+          inventoryInStock: variant.inventoryInStock || 0,
```

Comment:
This may not be the time to correct this, but `inventoryInStock` and `inventoryAvailableToSell` are optional in the catalog GraphQL schema, and it seems to me that `null` would mean something different from `0`. If inventory isn't tracked in the product (`null`) but yet that becomes `0` in the catalog, that seems to be changing the meaning from "unknown" to "none".

So essentially I'm proposing that we remove the `|| 0` from all of these